app/views/shared/_newsletter.slim

```diff
@@ -2,6 +2,6 @@ form action='//venturedeck.us5.list-manage.com/subscribe/post?u=79f7816bac08905f
   .Newsletter-input
     input.Input type='email' value='' placeholder='Your email address' name='EMAIL'
   .Newsletter-submit
-    input.ButtonPrimary.ButtonPrimary--fullWidth name='subscribe' type='submit' value='Get monthly updates'
+    input.ButtonPrimary.ButtonPrimary--fullWidth name='subscribe' type='submit' value='Get weekly updates'
```

Comment:
This input does not have a label. Since we are changing it, we might as well add it.